src/Automation/Automation/help/Register-AzAutomationDscNode.md

```diff
@@ -24,6 +24,12 @@ Register-AzAutomationDscNode -AzureVMName <String> [-NodeConfigurationName <Stri
 ## DESCRIPTION
 The **Register-AzAutomationDscNode** cmdlet registers an Azure virtual machine as an APS Desired State Configuration (DSC) node in an Azure Automation account.
 
+If you need to register a node to an automation account in a different subscription,
+you will need to use an ARM template rather than cmdlets. 
+See the Azure Automation
+[documentation](https://docs.microsoft.com/en-us/azure/automation/automation-dsc-onboarding#registering-virtual-machines-across-azure-subscriptions)
+for more details.
```

Comment:
Please add a changelog entry for this change in the Automation Changelog

Reply:
done